README.md

```diff
@@ -30,6 +30,23 @@ Conjur provides secrets management and application identity for modern infrastru
 
 _Note: our badges and social media buttons never track you._
 
+- [Community Support](#community-support)
+- [Migrating to Conjur EE](#migrating-to-conjur-ee)
+- [Architecture](#architecture)
+  * [Database](#database)
+    + [DATABASE_URL environment variable](#database-url-environment-variable)
+    + [Database initialization](#database-initialization)
+  * [Authenticators](#authenticators)
+  * [Rotators](#rotators)
+  * [Secrets and keys](#secrets-and-keys)
+    + [Important: avoid data loss](#important--avoid-data-loss)
+  * [Account management](#account-management)
+- [Versioning](#versioning)
+- [License](#license)
```

Comment:
Contributing link is not in this ToC. I think there may be other missing headers too.

Reply:
good catch!! fixed